crates/terminal_view/src/terminal_scrollbar.rs

```diff
@@ -0,0 +1,149 @@
+use std::ops::Range;
```

Comment:
Can we perhaps use a scrollbar from `ui` crate?

Reply:
Perhaps we could traitify it a bit, to make it easier for other crates (e.g. `terminal_view`) to provide their own definition of what a scrollable handle is.